pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go

```diff
@@ -256,6 +259,70 @@ func (im *Manager) evalTask(job *Job) error {
 	return nil
 }
 
+// deployTaskII starts deploy task when job supports hot model update
+func (im *Manager) deployTaskII(job *Job) error {
+	var localModelConfigFile string
+	if v, ok := job.ObjectMeta.Annotations[runtime.ModelHotUpdateAnnotationsKey]; ok {
+		localModelConfigFile = v
+	} else {
+		return nil
+	}
+
+	latestCond := im.getLatestCondition(job)
+	currentType := latestCond.Type
+
+	if currentType == sednav1.ILJobStageCondRunning && job.JobConfig.DeployTriggerStatus == TriggerReadyStatus {
+		models := im.getModelFromJobConditions(job, sednav1.ILJobDeploy)
+		if models == nil {
+			return nil
+		}
+		trainedModel := models[0]
+		deployModel := models[1]
+
+		trainedModelURL := trainedModel.URL
+		_, modelName := filepath.Split(trainedModelURL)
+		localHostDir, _ := filepath.Split(localModelConfigFile)
+		localHostModelFile := filepath.Join(localHostDir, modelName)
+
+		modelFile := util.AddPrefixPath(im.VolumeMountPrefix, localHostModelFile)
+		if err := im.updateDeployModelFile(job, trainedModelURL, modelFile); err != nil {
+			return err
+		}
+
+		deployModelURL := deployModel.URL
+		if err := im.updateDeployModelFile(job, trainedModelURL, deployModelURL); err != nil {
+			return err
+		}
+
+		config := map[string]map[string]string{
+			"model_config": {
+				"model_path": strings.Replace(localHostModelFile,
```

Comment:
why don't use relative path for `model_path` with config file instead of tweaking the absolute path?

Reply:
fix it